src/org/labkey/test/tests/list/ListTest.java

```diff
@@ -1196,23 +1273,27 @@ private void customFormattingTest()
     @Test
     public void doRenameFieldsTest()
     {
+        // Issue 8329
         log("8329: Test that renaming a field then creating a new field with the old name doesn't result in awful things");
         String listName = "new";
-        String origFieldName = "BarBar";
-        String newFieldName = "FooFoo";
+
+        // Issue 52480
+        String origFieldName = ": random 1 /@\".";
+        String newFieldName = "random 1";
```

Comment:
I might not use "random" in the names here lest we mistake these for names that are actually generated.